### ISSUE TITLE
Access to fetch blocked by CORS policy

### ISSUE THREAD

**Issue by godanny86** [AdobeDocs/app-builder#156]
Issue in /src/pages/getting_started/first_app.md

While trying to follow the above steps I get the following error in the browser when trying to run the `generic` action the first time when creating a new App Builder (formerly project firefly) app

```
Access to fetch at 'https://<host>/api/v1/web/dx-excshell-1/generic' from origin 'https://localhost:9080' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status.
```

**Steps to reproduce**

1. Follow all of the steps 1-5 listed on this page https://developer.adobe.com/app-builder/docs/getting_started/first_app/#6developing-the-application ( I used the dx-excshell template)
2. In step **6** run the command `aio app run`
3. Using the Ex-c Shell Navigate to **Your App Actions** > **Actions** > **generic** and tap **Invoke**
  
    ![image](https://user-images.githubusercontent.com/8974514/148983767-a3a76b5d-0841-4ad1-8da9-77f0be3d73cd.png)

4. Observe the CORS policy error

I believe this is because the `generic` actions has not been deployed and yields a 404 (instead of 200 ok)

If I first run the `aio app deploy` and then `aio app run` everything works as expected and I can continue to iterate on the application.

I think the steps listed in https://developer.adobe.com/app-builder/docs/getting_started/first_app/#6developing-the-application should be modified or at a minimum list the following in **Common Issues**

**Comment by MichaelGoberling** [comment]
This can be repro'd. I believe the issue is that following the tutorial results in the following order of operations: 

* `aio app run --local` deploys actions locally 
* `aio app run` doesn't perform the action deployment to Runtime, because there are no changes detected, making the following tutorial note technically incorrect

```
This will deploy the actions to [Adobe I/O Runtime](https://developer.adobe.com/apis/experienceplatform/runtime), while running the UI part on the local machine.
```

* Subsequent runs of `aio app run` won't do anything without code changes to actions, see 2
* `aio app deploy` will force the action deployment 

Will look into possibly flipping the `aio app run` and `aio app run --local` commands to see if that fixes this, but I believe the actions may not be deployed locally if we do that.

**Comment by MichaelGoberling** [comment]
Flipping the commands won't suffice, the assumption that actions wouldn't be deployed was true. 

The easiest way forward may be just to add an instruction to run `aio app deploy` as part of the tutorial before running `aio app run`, noting that this will deploy the actions to Runtime.

**Comment by shazron** [comment]
should be fixed: https://github.com/adobe/aio-cli-plugin-app/pull/551

**Comment by MichaelGoberling** [comment]
Closing as the above PR was included in app plugin release 9.0.0. If still experiencing this, try upgrading the aio cli to 9.0.0+
